docs/docs/evaluation/index.md

```diff
@@ -0,0 +1,218 @@
+# Evaluation CLI
+
+:::warning[👷‍♂️ Work In Progress 👷‍♂️]
+
+The Evaluation CLI documentation is under active development
+and will be expanded in the near future.
```

Comment:
Maybe we could link to the issues page here ("that said, if you see an issue or have a suggestion, ...")

Reply:
```suggestion
and will be expanded in the near future.

If you see an issue or have a suggestion, please [file an issue on Github](https://github.com/mongodb/chatbot/issues).
```